### PR TITLE
Go Module V2

### PR DESCRIPTION
**Pull request by appilon**
After this is merged, we can start tagging this branch with alphas/betas/release candidates.

### CODE REVIEW

File: helper/schema/field_reader_diff_test.go [hunk 1]
```diff
@@ -109,7 +109,7 @@ func TestDiffFieldReader_NestedSetUpdate(t *testing.T) {
 	}
 }
 
-// https://github.com/hashicorp/terraform-plugin-sdk/issues/914
+// https://github.com/hashicorp/terraform-plugin-sdk/v2/issues/914
```
Comment (paultyng) [inline helper/schema/field_reader_diff_test.go]: this is wrong

File: helper/schema/schema_test.go [hunk 2]
```diff
@@ -1949,7 +1949,7 @@ func TestSchemaMap_Diff(t *testing.T) {
 		},
 
 		{
-			Name: "https://github.com/hashicorp/terraform-plugin-sdk/issues/824",
+			Name: "https://github.com/hashicorp/terraform-plugin-sdk/v2/issues/824",
```
Comment (paultyng) [inline helper/schema/schema_test.go]: shouldn't be updated

File: helper/schema/shims_test.go [hunk 3]
```diff
@@ -2234,7 +2234,7 @@ func TestShimSchemaMap_Diff(t *testing.T) {
 		},
 
 		{
-			Name: "https://github.com/hashicorp/terraform-plugin-sdk/issues/824",
+			Name: "https://github.com/hashicorp/terraform-plugin-sdk/v2/issues/824",
```
Comment (paultyng) [inline helper/schema/shims_test.go]: shouldn't be updated

File: .go-version [hunk 4]
```diff
@@ -1 +1 @@
-1.12.9
+1.12.17
```
Comment (paddycarver) [inline .go-version]: Given we're on Go 1.14 now, should we bump the version accordingly?

Comment (appilon) [reply .go-version]: I think in general I held off go1.13 personally (for all my development) for some weird transitive dependency things... I can't remember them at all so sure let's move to go1.14!

**Pull request merged**